client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/bean/ParameterInfo.java

```diff
@@ -2,7 +2,8 @@
 
 import java.lang.reflect.Parameter;
 
-public class ParameterInfo {
+public class
+ParameterInfo {
```

Comment:
```suggestion
public class ParameterInfo {
```

Reply:
Or was it intentional?

Reply:
No, of course it was by accident, thanks